src/Tasks/Microsoft.NET.Build.Tasks/ProcessFrameworkReferences.cs

```diff
@@ -688,13 +703,13 @@ private enum ToolPackType
             }
 
             // ILCompiler and ILLink have RID-agnostic build packages that contain MSBuild targets.
-            if (toolPackType is ToolPackType.ILCompiler or ToolPackType.ILLink)
+            if (toolPackType is ToolPackType.ILCompiler or ToolPackType.ILLink or ToolPackType.WebAssemblySdk)
```

Comment:
Not a strong preference here but it seems like
```suggestion
            if (toolPackType is not ToolPackType.Crossgen2)
```
is slightly cleaner